XCUITests/TopTabsTest.swift

```diff
@@ -351,3 +351,21 @@ class TopTabsTestIphone: IphoneOnlyTestCase {
         XCTAssertTrue(app.buttons["TabTrayController.maskButton"].isEnabled)
     }
 }
+
+class TopTabsTestIpad: IpadOnlyTestCase {
+    func testUpdateTabCounter(){
```

Comment:
Before this line I think is necessary to add `if skipPlatform { return }` so that it only runs on iPad